### PR TITLE
Redesign action working mode

### PR DESCRIPTION
**Pull request by septs**
image

This new solution requires only one seconds of running time

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,29 +1,26 @@
-on: [push, pull_request]
-name: test
+name: Test
+
+on:
```
Comment (septs) [inline .github/workflows/test.yml:3]: https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/2

Comment (jandelgado) [reply .github/workflows/test.yml:3]: that solves one of the mysteries of gh-actions for me. thanks

**Comment by septs** [comment]
@jandelgado request merge the pr

**Comment by jandelgado** [comment]
well done, thanks.

**Comment by septs** [comment]
plz release next tag.

**Comment by jandelgado** [comment]
done

**Pull request merged**